### ISSUE TITLE
Game hangs with white screen when running game on secondary display

### ISSUE THREAD

**Issue by dethmuffin** [godotengine/godot#35013]
**Godot version:**
3.1.2.stable

**OS/device including version:**
Windown 10 Pro 10.0.177643 Build 17763
OpenGL ES 3.0 Renderer: GeForce GTX 1080 with Max-Q Design/PCIe/SSE2

**Issue description:**
Game Run hangs with a white screen when running on a secondary screen "Monitor 2". Killing the screen. results in:
** Debug Process Started **
 drivers/unix/net_socket_posix.cpp:190 - Socket error: 10054
** Debug Process Stopped 

Runs fine on primary display.

**Steps to reproduce:**
1. Set Editors Settings-General->Run->Window Placement->Screen to Monitor 2
2. Run any project.
3. Observe hang with white screen and error when window is killed.

If you set step 1. to Monitor 1 the game runs as expected.

100% on my machine.

**Minimal reproduction project:**
You can use any project.

**Comment by Calinou** [comment]
Note that "Socket error: 10054" just means the game process crashed and resulted in the editor losing the debugger connection to the game.

**Comment by dethmuffin** [comment]
Sorry, no it does not happen the other way around on my machine. Thanks!

**Comment by carlholmes** [comment]
I also have this behavior with Godot_v3.2.1-stable_win64.exe running on Windows 8.1.  

I downloaded this over 3.2 because I had this issue with just trying to follow along with the simple "Hello World" in the docs - I had the documentation open on my primary monitor and the editor in the secondary editor and running the scene resulted in a white application popup.  

So, I immediately downloaded the update and tried again - but the update defaulted to launch with my main monitor, and it launched the scene just fine.  But then I moved it back to my secondary monitor, and attempted to launch the scene, and received the same white box ("non-responsive program").  This was surprising to someone who's new with Godot to just have "Hello World" not work ;)  Thankfully, I can default the running to my main monitor for now, so I'll do just that.
